composer.json

```diff
@@ -10,13 +10,13 @@
         "cweagans/composer-patches": "~1.0",
         "drupal/console": "~0.10",
         "drupal/core": "~8.1.0",
-        "drupal/facets" : "~8.1.0-alpha3",
+        "drupal/facets" : "8.1.0-alpha3",
```

Comment:
The leading tilde is missing. If this is omitted we will forever be stuck on this alpha version :)